packages/components/components/elvis-app-bridge/src/react/AppIcon.tsx

```diff
@@ -0,0 +1,60 @@
+import { getThemeColor } from '@elvia/elvis-colors';
+import { getTypographyCss } from '@elvia/elvis-typography';
+import styled from '@emotion/styled';
+import React from 'react';
+
+const IconContainer = styled.div`
+  position: relative;
+  border-radius: 50%;
+  width: 24px;
+  height: 24px;
+  overflow: hidden;
+  object-fit: fill;
+`;
+
+const Icon = styled.div`
+  position: absolute;
+  width: 100%;
+  height: 100%;
+
+  svg {
+    width: 100%;
+    height: 100%;
+  }
+`;
+
+const IconLetters = styled.div`
+  ${getTypographyCss('text-sm-strong')};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: ${getThemeColor('static-white')};
+
+  position: relative;
+  height: 100%;
+`;
```

Comment:
I have not tested this; would it be easier to use a CSS grid and place all items into the same cell? 🤔

```css
.icon-container {
  display: grid;
  place-content: center;

  > * {
    grid-area: 1 / 1;
  }
}
```

Reply:
This is just about a pure copy of the way it's done in the Header component, maybe grid is cleaner but 🤷